### ISSUE TITLE
Blocks and definition of sets

### ISSUE THREAD

**Issue by p-snft** [oemof/oemof#579]
According to the documentation, for each component (-block) a corresponding set is created (see e.g. [solph.blocks.Transformer](https://oemof.readthedocs.io/en/v0.2.3/api/oemof.solph.html#oemof.solph.blocks.Transformer). However, I cannot find this implemented in the code. For some blocks (e.g. [GenericStorage](https://oemof.readthedocs.io/en/v0.2.3/api/oemof.solph.html#oemof.solph.components.GenericStorageBlock)), however, I do find the code creating the set.

Is this an error in the documentation or should the sets be created? (Obviously, they are not needed at the moment - but there might be a reason to have them available.)

**Comment by uvchik** [comment]
According to #544 we should talk about a consistent docstring format at the oemof meeting and I really would like to remove the sets from the docstring.

**Comment by simnh** [comment]
Actually It might be that there are no specific pyomo sets for each block. But at least there will be a group returned by the the grouping functionality which is used by every component in constraint_group method that returns a set which relates the component to its block. So there is a set for each block, even though its in the es.groups attribute.

Nevertheless there are the sometimes multiple set created within on block, which are kind of helping set for construction of constraints. 

In short I think its not really wrong but also not very specific and misleading. A more correct description would be great.

**Comment by p-snft** [comment]
Defined sets are no longer part of the docstring template (#544). So, this kind of confusion will be avoided in the future.
